HealthcareSystem/Backend/Service/NotificationSurveyAndFeedback/FeedbackService.cs

```diff
@@ -13,22 +14,33 @@ namespace Service.NotificationSurveyAndFeedback
 {
    public class FeedbackService
    {
-        private FeedbackRepository feedbackRepository = new FeedbackRepository();
-        public void NewFeedback(Feedback feedback)
+        private IFeedbackRepository _feedbackRepository;
+
+        private ActivePatientRepository _activePatientRepository = new ActivePatientRepository();
+
+        public FeedbackService(IFeedbackRepository feedbackRepository)
+        {
+            _feedbackRepository = feedbackRepository;
+        }
+        public void AddFeedback(Feedback feedback)
         {
-            feedbackRepository.NewFeedback(feedback);
+            if (feedback == null)
+                return;
```

Comment:
I would throw exceptions here instead, this way the caller doesn't know if the operation was successful or not. Maybe now would be a good moment to add custom exceptions?